java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -107,10 +107,11 @@ public static void assertPartialStructColumnsAreEqual(ColumnVector expected, lon
   }
 
   public static void assertPartialColumnsAreEqual(ColumnVector expected, long rowOffset, long length,
-                                                  ColumnVector cv, String colName, boolean enableNullCheck) {
+                                                  ColumnVector cv, String colName, boolean enableNullCheck,
+                                                  HostColumnVector.ColumnBuilder.DataType type) {
```

Comment:
Can we please make `type` a little clearer?  In other places it is called schema.  Also at this point it might be good to have some javadocs to explain what is happening here.  Honestly we should have done it a lot sooner.